### PR TITLE
Move hack dirs to scripts dir

### LINKED ISSUE #293: [Feature] Restructure the hack directories
### Search before asking

- [X] I searched the [issues](https://github.com/IBM/data-prep-lab/issues) and found no similar issues.

### Component

Other

### Feature

We have the `transformers/hack` directory, which contains a single script, we need to move it to the `scripts` directory.
Check the possibility to do the same with the `kind/hack` 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by revit13**
/Closes #293

This PR moves the hack subdirs under a new `scripts` directory

## Related issue number (if any).

#293

### CODE REVIEW

**Review by roytman — APPROVED** [review]
LGTM

File: scripts/kind/tools/ingress.sh [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 op=$1
 
-source ${ROOT_DIR}/hack/common.sh
+source ${REPOROOT}/scripts/kind/common.sh
```
Comment (roytman) [inline scripts/kind/tools/ingress.sh]: can you define a local variable `{REPOROOT}/scripts/kind`, we have too many references to it.

File: scripts/kind/tools/ingress.sh [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 
 deploy() {
-  kubectl apply -f ${ROOT_DIR}/hack/ray_api_server_ingress.yaml
+  kubectl apply -f ${REPOROOT}/scripts/kind/ray_api_server_ingress.yaml
```
Comment (roytman) [inline scripts/kind/tools/ingress.sh]: can you add sleep 10 before L:8

File: transforms/.make.workflows
Comment (daw3rd) [inline transforms/.make.workflows:1]: This is an inconsistent move.  A) it is not a shell script and B) if we do this, we should also move .make.defaults, .make.versions,.make.transforms etc.  But I'm not sure we want to do this.  What was your motivation here?

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: transforms/.make.workflows
Comment (revit13) [reply transforms/.make.workflows:1]: @daw3rd transforms/.make.workflows file remains under transforms... are you refering to other file?

File: scripts/kind/tools/ingress.sh (hunk 2 above)
Comment (revit13) [reply scripts/kind/tools/ingress.sh]: Added. Thanks

File: scripts/kind/tools/ingress.sh (hunk 1 above)
Comment (revit13) [reply scripts/kind/tools/ingress.sh]: fixed. Thanks

**Review by daw3rd — APPROVED** [review]
lgtm

**Pull request merged**